### DISCUSSION TITLE
How to use nodejs tailwind?

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#5876] [Help]
Hello,
So on the website it says that tailwind is about 600kb but because of some sort of a minifier it removes unused classes.

I'm working with a SPA that is self written not React etc, can I still use tailwind with NodeJS?

If yes how do I minify the file so it deletes the unused classes?

**Comment by NovusOpus** [comment]
You can use the Tailwind CLI, Here's a video from Tailwind CSS team that explains how to: [https://youtu.be/DxcJbrs6rKk?t=14](https://youtu.be/DxcJbrs6rKk?t=14). Or you can check the docs page: [Tailwind CLI](https://tailwindcss.com/docs/installation#using-tailwind-cli).

But if you want to try something better on the fly for testing purposes, the new version of Tailwind CSS (Version 3, Currently in Alpha) is coming soon, and comes with a new feature:  Just-in-Time CDN.

Quoting from the announcement:


> It lets you use every feature Tailwind CSS has to offer, and does it in 93kB — which ain't bad for something originally designed to run in node with no file size considerations at all.

You can read more about it here: [Tailwind CSS v3.0.0-alpha.1 Announcement](https://github.com/tailwindlabs/tailwindcss/discussions/5668)

With Just-in-Time CDN you will be able to just throw this tag: `<script src="https://cdn-tailwindcss.vercel.app/"></script> ` in your `<head>` 

Please Note that: 

> The Just-in-Time CDN is intended for development purposes only, and should not be used in production.
